Makefile

```diff
@@ -103,6 +105,18 @@ LANTERN_YAML_PATH := installer-resources/lantern.yaml
 
 .PHONY: packages clean tun2socks android-lib android-sdk android-testbed android-debug android-release android-install docker-run
 
+define require-node
```

Comment:
With the new UI we won't need node or npm at all from the Makefile @oxtoacart, but we can handle that in the merge.